webpack/src/Extends/Host/PuppetTab/SubTabs/YamlTab/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useAPI } from 'foremanReact/common/hooks/API/APIHooks';
+import PropTypes from 'prop-types';
+
+import Skeleton from 'react-loading-skeleton';
+import EmptyState from 'foremanReact/components/common/EmptyState/EmptyStatePattern';
+import { STATUS } from 'foremanReact/constants';
+import { EmptyStateIcon } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import { global_danger_color_200 as dangerColor } from '@patternfly/react-tokens';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { YamlData } from './YamlData';
+
+const YamlTab = ({ hostName }) => {
```

Comment:
can we change this to `ENCTab` or `ENCPreview`? the directory should be renamed as well

Reply:
Yes, makes sense. I changed it to `ENCPreview` to be more consistent with the Tab name.